README.md

```diff
@@ -1,5 +1,5 @@
 # Go Stripe
-[![Go Reference](https://pkg.go.dev/badge/github.com/stripe/stripe-go)](https://pkg.go.dev/github.com/stripe/stripe-go)
+[![Go Reference](https://pkg.go.dev/badge/github.com/stripe/stripe-go)](https://pkg.go.dev/github.com/stripe/stripe-go/v74)
```

Comment:
Should we add this to [the Makefile target](https://github.com/stripe/stripe-go/blob/master/Makefile#L37) ?

Reply:
I thought [this line in the Makefile](https://github.com/stripe/stripe-go/blob/master/Makefile#L37) already updates the version. For example, when I run `make update-version VERSION="73.0.0"`, it updates the path to `https://pkg.go.dev/github.com/stripe/stripe-go/v73`.